Makefile

```diff
@@ -12,3 +12,9 @@ clean-pyc:
 run: 
 	poetry install
 	poetry run python main.py
+
+.PHONY: fmt
+fmt:
+	poetry run isort nodestream_akamai
+	poetry run black nodestream_akamai
```

Comment:
added this because I failed an automated PR check, so I felt I should make it easier for me to do the right thing in the future